src/app/state/accountHistorySlice.ts

```diff
@@ -31,6 +31,10 @@ export interface AccountHistoryState {
   selectedTable: Tables;
   tables: Tables[];
   selectedOrdersToCancel: Record<string, Order>; // the key is `${orderRecieptAddress}_${nftRecieptId}`
+  hideOtherPairs: boolean;
+  orderHistoryAllPairs: adex.OrderReceipt[];
+  error: string | null;
```

Comment:
You could make the error optional so you don't have to initialize it
```suggestion
  error?: string;
```